examples/src/pages/Clipping.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright 2020 Cognite AS
+ */
+import React, { useEffect, useRef } from "react";
+import * as THREE from "three";
+import * as reveal from "@cognite/reveal/experimental";
+import CameraControls from "camera-controls";
+import dat from "dat.gui";
+import {
+  getParamsFromURL,
+  createRenderManager,
+} from "../utils/example-helpers";
+import { CogniteClient } from "@cognite/sdk";
+import { BoundingBoxClipper } from "@cognite/reveal";
+import { CanvasWrapper } from "../components/styled";
+
+CameraControls.install({ THREE });
+
+export function Clipping() {
+  const canvasRef = useRef<HTMLCanvasElement>(null);
+
+  useEffect(() => {
+    const gui = new dat.GUI();
+
+    async function main() {
+      const { project, modelUrl, modelRevision } = getParamsFromURL({
```

Comment:
I know that it might not be part of this PR, since you say you don't change much, but would it be possible if at least one example could be changed to fit a more React style of setup as an example on how it would look that way?

The component should be provided parameters from their parents where possible e.g. client, modelUrl / modelRevision. Thoughts?

Right now it seems like we are using react, but not really taking advantage of it. I could be mistaken but the react router makes chaging url not actually reload pages, and in that sense we should structure the examples more like an actual application?

Reply:
> Right now it seems like we are using react, but not really taking advantage of it.

Indeed. I wanted to refactor at least one of the examples to be more 'reactish' but later. 
About URL params, they are still work as they worked before. You enter params, hit enter, page reloads, params parsed. 

> The component should be provided parameters from their parents where possible e.g. client, modelUrl / modelRevision. Thoughts?

Not really necessary IMO but doable. My concerns here are that different examples have different params so you can't just put URL params upper in components hierarchy and pass them to children in the same way for all examples.